spec/queries/get_fraud_matches_spec.rb

```diff
@@ -4,38 +4,103 @@
   let(:candidate1) { create(:candidate, id: '1', email_address: 'exemplar1@example.com') }
```

Comment:
Having hard-coded `id`s is a bit weird. You could remove and just assert on `candidate1.id` etc.

Reply:
Nice one, will push up a fix!